homeassistant/components/edilkamin/config_flow.py

```diff
@@ -0,0 +1,98 @@
+"""Config flow for edilkamin integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import edilkamin
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN, LOGGER
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
```

Comment:
Use `cv.str` instead of `str`. Import from:

`import homeassistant.helpers.config_validation as cv`

Reply:
Addressed, I didn't know.
I'm wondering why the scaffolding script doesn't use it :man_shrugging: 
Do we know?
https://github.com/home-assistant/core/blob/2022.12.1/script/scaffold/templates/config_flow/integration/config_flow.py#L21-L23